Makefile

```diff
@@ -209,20 +209,28 @@ gitserver:
 		-t $(GITSERVER_IMAGE) \
 		--push .
 
+# Pipelines and ISBs are deleted first as they may have finalizers which will cause delete all to hang
```

Comment:
```suggestion
# Pipelines and ISBs are deleted first as they may have finalizers which will cause "delete all" to hang (because if a Controller is also being deleted, it first needs to remove the finalizers before it's deleted)
```